app/src/main/java/com/example/caregiver/ViewProgress.java

```diff
@@ -10,6 +10,41 @@
 
 public class ViewProgress extends AppCompatActivity {
 
+    enum TaskStatus {Completed, Incomplete, InProgress};
+
+    public class Task {
+
+        // TODO: Eventually this will hold data from Firebase representing a task. For now this is a
+        // dummy class so I can build a ListViewAdapter around it.
+
+        private String name;
+        private Duration timeCompleted;
+        private TaskStatus status;
+
+        public Task(String name, Duration timeCompleted, TaskStatus status)

```

Comment:
@zmcconnell02 Create an issue for Zak to use the task object in the model instead of this Task object.